board/iCE40CW312/info.yml

```diff
@@ -0,0 +1,5 @@
+Label: iCE40CW312
+Device: ICE40-UP5K
+Package: UWG30
+Documentation:
+  "gh:newaetech/chipwhisperer-target-cw308t/CW312T_ICE40UP": https://github.com/newaetech/chipwhisperer-target-cw308t/CW312T_ICE40UP
```

Comment:
This is a very quick review, since I have to leave no. I will check it later.

Nonetheless, the target URL needs to be fixed here. You need to add `tree/master` between the repo and the subdir: https://github.com/newaetech/chipwhisperer-target-cw308t/tree/master/CW312T_ICE40UP